### PR TITLE
Add page for alerts

### LINKED ISSUE #314: Add page about alerts

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #314

### CODE REVIEW

**Comment by johnsudaar** [comment]
@EtienneM is it still valid after all the dashboard modifications?

**Comment by EtienneM** [comment]
You are right, I must take new screenshots

**Comment by EtienneM** [comment]
https://documentation-service-pr315.scalingo.io/platform/app/alert

**Comment by EtienneM** [comment]
OK but I don't really see what can be different on this matter

File: _posts/platform/app/2000-01-01-alerts.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  1. RPM: requests per minute (RPM) received by your application.
+  1. RPM per container: If your application is scaled on multiple containers, the RPM per container
+     divides the RPM of the application by the amount of containers.
+- Limit: any float value.
+- Send when below: send the alert when metric value is _below_ the limit.
+- Notifiers: list of [notifiers]({% post_url platform/app/2000-01-01-notification %}) the alert send
+  a notification to.
+
+{% note %}
+All the metrics usable to define a new alert are the same available in the [metrics tab]({% post_url
```
Comment (johnsudaar) [inline _posts/platform/app/2000-01-01-alerts.md]: No true for the RPM per container

**Pull request merged**